### PR TITLE
Remove graph-app-kit dependency and upgrade sample graph imports to s…

### PR DESCRIPTION
**Pull request by tomasonjo**
Remove graph-app-kit dependency
Upgrade sample graphs to support both v4 and v5 Cypher syntax

### CODE REVIEW

File: src/components/SampleGraphs/sampleGraphs.js [hunk 1]
```diff
@@ -1,44 +1,47 @@
...
-        queries: [
-            `CREATE CONSTRAINT ON (p:Place) ASSERT p.name IS UNIQUE`,
-            `USING PERIODIC COMMIT 1000
+      name: "European Roads",
+      author: "Lasse Westh-Nielsen",
+      authorLink:
+        "http://lassewesth.blogspot.com/2018/07/the-international-e-road-network-and.html",
+      description: `A dataset containing European Roads.`,
+      queries: [
+        `CREATE CONSTRAINT ON (p:Place) ASSERT p.name IS UNIQUE`,
```
Comment (jexp) [inline src/components/SampleGraphs/sampleGraphs.js:43]: is this still correct syntax?

File: src/components/SampleGraphs/sampleGraphs.js [hunk 2]
```diff
@@ -49,28 +52,30 @@ MERGE (destination:Place {name: row.destination_reference_place})
...
-        description: `A dataset containing Twitter followers of the graph community`,
-        queries: [`CREATE CONSTRAINT ON(u:User) ASSERT u.id IS unique`,
-            `CALL apoc.load.json("https://github.com/neo4j-apps/neuler/raw/master/sample-data/twitter/users.json")
+    Twitter: {
+      name: "Twitter",
+      author: "Mark Needham",
+      authorLink: "https://markhneedham.com/blog/",
+      description: `A dataset containing Twitter followers of the graph community`,
+      queries: [
+        `CREATE CONSTRAINT ON(u:User) ASSERT u.id IS unique`,
```
Comment (jexp) [inline src/components/SampleGraphs/sampleGraphs.js:77]: constraint syntax?

Comment (tomasonjo) [reply src/components/SampleGraphs/sampleGraphs.js:77]: Should be fine as this is 4.0 syntax, while 5.x syntax is https://github.com/neo4j-devtools/neuler/blob/0d83eb767ba81640ff357246a4905b6e733153f6/src/components/SampleGraphs/sampleGraphs.js#L285

File: src/components/SampleGraphs/sampleGraphs.js (hunk 1 above)
Comment (tomasonjo) [reply src/components/SampleGraphs/sampleGraphs.js:43]: similar as the other, this is 4.x syntax, while 5.x is https://github.com/neo4j-devtools/neuler/blob/0d83eb767ba81640ff357246a4905b6e733153f6/src/components/SampleGraphs/sampleGraphs.js#L251

**Pull request merged**